app/components/primer/clipboard_copy.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if content.present? %>
     <%= content %>
```

Comment:
We'll need to check for cases where we pass in content that uses the paste icon.

Reply:
Would you like me to do that in this PR?

Reply:
Sorry, I meant when we go to add this change to dotcom. I just wanted to call our that there may be uses of this component that pass in the `paste` icon as content.